src/radio/sx126x/radio.c

```diff
@@ -983,6 +983,7 @@ void RadioSetRxDutyCycle( uint32_t rxTime, uint32_t sleepTime )
 
 void RadioStartCad( void )
 {
+    SX126xSetDioIrqParams( IRQ_RADIO_ALL, IRQ_RADIO_ALL, IRQ_RADIO_NONE, IRQ_RADIO_NONE );

```

Comment:
Instead of 
```c
SX126xSetDioIrqParams( IRQ_RADIO_ALL, IRQ_RADIO_ALL, IRQ_RADIO_NONE, IRQ_RADIO_NONE );
```
I would propose:
```c
SX126xSetDioIrqParams( IRQ_CAD_DONE | IRQ_CAD_ACTIVITY_DETECTED , IRQ_CAD_DONE, IRQ_RADIO_NONE, IRQ_RADIO_NONE );
```

Could you please check this proposal.

Reply:
I choose to enable all IRQ exactly as it's done in RX mode. 
I don't know the SX126x enough to be sure if it's relevant. 
As far as I know, CAD is a RX like mode

Reply:
Yes, the CAD is a simplified Rx mode. The receiver is not fully enabled in order to reduce the power consumption.

I have checked with my colleagues and my proposition is supposed to work. 
I let you verify the proposition on your system and then update this pull-request.

Reply:
resolved with commit 805b5c0